### PR TITLE
Update time conductor inputs realtime

### LINKED ISSUE #4352: Scrubbing and zooming the time conductor should update the start and end inputs in real time
#### Summary
When scrubbing or zooming the time conductor, the values in the start and end inputs should update in realtime. Note that the bounds in the time API should **not** update until the user has released the mouse.

This is a very recent regression and might be related to the [independent time conductor changes](https://github.com/nasa/openmct/pull/3988).

Screen Recording 2021-10-22 at 1 50 01 PM

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [X] Regression? Did this used to work or has it always been broken?
- [ ] Is there a workaround available?
- [X] Does this impact a critical component?
- [ ] Is this just a visual bug?

#### Steps to Reproduce
1. Switch to fixed time mode
2. Hold the option key (mac) and drag the time conductor axis left and right
3. Observe that the input values for the start and end time do not change until the mouse is released

#### Environment
* Open MCT Version: 1.7.8
* OS: All
* Browser: All

### PR DESCRIPTION
**Pull request by shefalijoshi**
Closes #4352

### Describe your changes:
Send in updated bounds to fixed and realtime inputs from the Conductor component
2.6.x changed how v-on works. So we need to downgrade to 2.5.x and later update Vue when we assess all the stuff that's broken and fix it in the entire app.

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Commit messages meet standards?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

**Comment by jvigliotta** [comment]
@nikhilmandlik to take a look.

File: package.json [hunk 1]
```diff
@@ -63,10 +63,10 @@
     "sinon": "13.0.1",
     "style-loader": "^1.0.1",
     "uuid": "^3.3.3",
-    "vue": "2.6.14",
+    "vue": "2.5.6",
```
Comment (nikhilmandlik) [inline package.json:66]: any specific reason for downgrade?

**Review by nikhilmandlik — COMMENTED** [review]
just curious about vue downgrade else LGTM.
Also CI failed 😅

**Review by nikhilmandlik — APPROVED** [review]

**Comment by shefalijoshi** [comment]
We're going to keep the 2.6.x version of Vue and try to fix what's broken. As such, this PR will be updated after approval.

**Review by nikhilmandlik — APPROVED** [review]
LGTM

**Pull request merged**